assets/src/edit-story/components/panels/panel/handle.js

```diff
@@ -33,24 +33,34 @@ import useDragHandlers from './useDragHandlers';
 import useKeyboardHandlers from './useKeyboardHandlers';
 
 const Handle = styled.button.attrs( { type: 'button', role: 'separator' } )`
-	background: transparent;
+	background-color: ${ ( { theme } ) => theme.colors.fg.v5 };
 	border: 0;
 	padding: 0;
-	height: 10px;
+	height: 6px;
 	display: flex;
 	flex-direction: column;
-	justify-content: flex-start;
+	justify-content: center;
 	align-items: center;
 	cursor: row-resize;
+
+	&:focus,
```

Comment:
Can we at least keep the focus styles for accessibility?

Reply:
How do we feel about using this https://github.com/lindsayevans/outline.js ?

Reply:
We don't have to keep outlines, but there should be some sort of focus style, e.g. by changing the bar's color to purple on focus.

Reply:
Didn't look so great so I took the approach of only applying `outline` with keyboard focus. [This](https://caniuse.com/#search=%3Afocus-visible) would've been great if browser supported it.